packages/textInput/components/TextInputWithIcon.tsx

```diff
@@ -6,7 +6,8 @@ import { TextInput, TextInputProps } from "./TextInput";
 import {
   getInputAppearanceStyle,
   inputContainer,
-  getIconAppearanceStyle
+  getIconAppearanceStyle,
+  inputIconWrapper
```

Comment:
The changes in this file are not related to the text input with buttons.

I noticed that icons could be set to any size when passed into the input, and that leaves us open to visual inconsistency. `inputIconWrapper` limits the size